### DISCUSSION TITLE
fatal error: 'CGAL/Simple_cartesian.h' file not found

### DISCUSSION THREAD

**Discussion by zhaoyuanyuan2011** [CGAL/cgal#7052] [Q&A] [answered]
## Issue Details

I've successfully installed CGAL, both from source, using brew or conda. However, I'm not able to include it in my cpp code. I pasted the error message and related CMakeList.txt part as follows. Thank you in advance for any idea!

## Source Code

In the CMakeList.txt, I specify the dirs like this:

```
set(CGAL_DIR /usr/local/lib/cmake)
find_package(CGAL CONFIG REQUIRED)
message(STATUS "Found cgal v${CGAL_VERSION}: ${CGAL_INCLUDE_DIRS}")
message(STATUS "cgal dir: ${CGAL_DIR}")
```

and during `cmake .` it gives me 

> -- Found cgal v5.5.1: /usr/local/include
> -- cgal dir: /usr/local/lib/cmake/CGAL

Everything looks fine, but I just get the error message when I'm trying to include CGAL in my code and run `make`

> [ 50%] Building CXX object CMakeFiles/mesh_hole_fill_eric.dir/mesh_hole_fill_eric_c2.cpp.o
> /Users/yzhao/Desktop/Docomo/small_point_cloud_completion/hole_fixer/biharmonic_r5/cpp_hole_filling_eric/mesh_hole_fill_eric_c2.cpp:21:10: fatal error: 'CGAL/Simple_cartesian.h' file not found
> #include <CGAL/Simple_cartesian.h>
>          ^~~~~~~~~~~~~~~~~~~~~~~~~
> 1 error generated.
> make[2]: *** [CMakeFiles/mesh_hole_fill_eric.dir/mesh_hole_fill_eric_c2.cpp.o] Error 1
> make[1]: *** [CMakeFiles/mesh_hole_fill_eric.dir/all] Error 2
> make: *** [all] Error 2

## Environment

* Operating system: Mac, 64 bits
* Compiler: Using /Library/Developer/CommandLineTools/usr/bin/c++ compiler
* Release mode
* CGAL version: 5.5.1
* Boost version: 1.74.0
* Other libraries versions if used: Eigen
* I also used no environments or python venv and conda environments but none of them work.

**Comment by lrineau** [comment, marked as answer]
In your `CMakeLists.txt`, after the definition of the target `mesh_hole_fill_eric`, add a line like that one:
```cmake
target_link_libraries(mesh_hole_fill_eric PRIVATE CGAL::CGAL)
```

See the documentation at https://doc.cgal.org/latest/Manual/devman_create_and_use_a_cmakelist.html.

**Reply by zhaoyuanyuan2011** [reply]
Thank you!!
